finance-portal/templates/secrets.yaml

```diff
@@ -21,3 +21,18 @@ data:
   JASPER_USER: {{ .Values.config.jasperUser | b64enc }}
   JASPER_PASSWORD: {{ .Values.config.jasperPassword | b64enc }}
 
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "finance-portal.fullname" . }}-certs
+  labels:
+    app.kubernetes.io/name: {{ include "finance-portal.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+data:
+  first_key: {{ required "The JWS key is required to validate the JWS header" .Values.creds.first_key | b64enc }}
+  second_key: {{ required "The JWS key is required to validate the JWS header" .Values.creds.second_key | b64enc }}
+  third_key: {{ required "The JWS key is required to validate the JWS header" .Values.creds.third_key | b64enc }}
```

Comment:
Should probably use a range, something like:
```
{{- range $k, $v =: .Values.JWSKeys }}
  {{ $k }}: {{ required "blah blah" $v }}
{{- end }}
```
Also, are you sure you need to `b64enc` these? I suspect k8s does that for you.

Reply:
`b64enc` is necessary for the secret to be installed into the k8s cluster. if it's not added, helm will output different types of error.